articles/install-command-line-qdk.md

```diff
@@ -30,7 +30,7 @@ Configure the QDK for your preferred environment from one of the following optio
 
 ### [VS Code](#tab/tabid-vscode)
 
-1. Download and install [VS Code](https://code.visualstudio.com/download) 1.52.0 or greater (Windows, Linux and Mac).
+1. Download and install [VS Code](https://code.visualstudio.com/download) 1.52.0 or greater (Windows, Linux and Mac). You can install VS Code through a graphical software installer or through the command line in your Linux environment.
```

Comment:
Does this provide new information? I'm assuming graphical software installers or command line package managers are generally the options available to you, although this could even depend on your distribution as some might not ship with a graphical software installer.

Reply:
I agree, this new line doesn't contribute anything new. I believe Linux users already know these things. The fewer instructions to read, the better.